src/test/java/uk/gov/hmcts/reform/darts/controllers/AuthInternalUserControllerTest.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.darts.controllers;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.web.servlet.ModelAndView;
+import uk.gov.hmcts.reform.darts.controllers.authentication.AuthInternalUserController;
+
+import java.util.Objects;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class AuthInternalUserControllerTest {
+
+    private final AuthInternalUserController authInternalUserController = new AuthInternalUserController();
+
+    @Test
+    void loginandrefreshtokenfromazureWhentokentoesntexistinsession() {
```

Comment:
These test names are quite hard to read. I think the general pattern on HMCTS is to use camel case for test names

Reply:
Well the ideal way is to use <methodName>_<expectedResult>_<condition> but the issue is Sonar criteria are bit messed up. I did use Camel casing but it only allowed me to use 2 upperCase test method name.